manifests/p/Prometheus/WindowsExporter/0.30.0/Prometheus.WindowsExporter.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with komac v2.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Prometheus.WindowsExporter
+PackageVersion: 0.30.0
+PackageLocale: en-US
+Publisher: Prometheus Community
+PublisherUrl: https://github.com/prometheus-community
+PublisherSupportUrl: https://github.com/prometheus-community/windows_exporter/issues
+PackageName: windows_exporter
+PackageUrl: https://github.com/prometheus-community/windows_exporter
+License: MIT
+LicenseUrl: https://github.com/prometheus-community/windows_exporter/blob/master/LICENSE
+Copyright: Copyright (c) 2016 Martin Lindhe
```

Comment:
I recently update the upgrade note, the current holder are `Copyright (c) 2021 The Prometheus Authors`. It's already part of all source code headers, but it seems to be forgotten on the LICENSE file.

https://github.com/prometheus-community/windows_exporter/blob/faa98d27087af1035d993074b444fec34e9d957b/LICENSE#L4

Reply:
```suggestion
Copyright: Copyright (c) 2021 The Prometheus Authors
```